Assets/HoloToolkit/UX/Scripts/Lines/LineUnity.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using HoloToolkit.Unity;
+using System.Collections;
+using UnityEngine;
+
+namespace MRTK.UX
+{
+    public class LineUnity : LineRendererBase
+    {
+        const string DefaultLineShader = "Particles/Alpha Blended";
+        const string DefaultLineShaderColor = "_TintColor";
```

Comment:
missing access modifiers

Reply:
Not needed since they're const.
I could make them static with a Get accessor? eg:
public static string DefaultLineShader { get { return "Particles / Alpha Blended"; } }

Reply:
So what happens if someone tries to use a custom shader for line rendering?

Reply:
These are just failsafe values. If they specify a material and property name, that material's shader is used. If they leave the material null, a material is generated at runtime using the "Particles/Alpha Blended" shader.